### ISSUE TITLE
Q: 404 fallback

### ISSUE THREAD

**Issue by jelen07** [nginx/unit#400]
👋 Hi, is there some option, how to configure routes to process static files via NGINX Unit and if the file does not exist, then process a script?

I was using this old one configuration in the Apache web server and looking for an alternative in the Unit.

```
# Images
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule ^(images\/)(.+) index.php [L]
```

At the moment, I'm seeing just `Error 404.` message.

**Comment by mar0x** [comment]
Hello,
Not yet, but it is planned in a coming release. Keep tracking the #392 .

**Comment by dbuelow** [comment]
Can be closed because the feature is present since v1.16.0.
